tests/health/fail.cmake

```diff
@@ -18,6 +18,6 @@ set(TOIT_FAILING_TESTS
 
 if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows" OR "${CMAKE_SYSTEM_NAME}" STREQUAL "MSYS")
   list(APPEND TOIT_FAILING_TESTS
-    tests/lock_file/multi_test/target/sub/src/sub.toit
+    health-tests/lock_file/multi_test/target/sub/src/sub.toit
```

Comment:
What's the easiest of reading this? Maybe split the prefix (the stuff controlled by the test infrastructure) and the stuff controlled by the external package, so it is easier to find the right stuff.

$(HEALTH_PREFIX)/sub/src/sub.toit

Reply:
Added a HEALTH_TEST_PREFIX in https://github.com/toitlang/toit/pull/383